api/httpjson/RPCserver.go

```diff
@@ -55,9 +55,6 @@ func NewServer(node protocol.Noder, wallet vault.Wallet) *RPCServer {
 }
 
 func (s *RPCServer) write(w http.ResponseWriter, data []byte) {
```

Comment:
This function is no longer required. Please remove this function and change `s.write(w, data)` to `w.Write(data)` is better.

Reply:
Did not realise, done!